.github/workflows/cli-publish.yml

```diff
@@ -24,20 +24,8 @@ jobs:
           tag_name: "cli-wallet-v${{ env.CRATE_VERSION }}"
           release_name: "cli-wallet-v${{ env.CRATE_VERSION }}"
           body: |
-            # Changelog
-
-            <!---  ## Added -->
-
-            <!---  ## Changed -->
-
-            <!---  ## Deprecated -->
-
-            <!---  ## Removed -->
-
-            <!---  ## Fixed -->
-
-            # Checksums
-
+            https://github.com/iotaledger/iota-sdk/blob/develop/cli/CHANGELOG.md
```

Comment:
Can we make this link to the specific tag? Otherwise on newer release the link wont "work" for old ones

Reply:
Not sure that's really worth it, a direct link would have to include tag+date which is not vert convenient

Reply:
We can create an issue for it if you want. I wouldn't do it here because we have the same for the other releases.